### PR TITLE
Prevent overscroll for amp-story

### LINKED ISSUE #11641: Prevent overscroll for <amp-story standalone>
We should prevent overscrolling, where users will see whitespace at the top/bottom of the document by scrolling past the document bounds. We can achieve this by setting `overflow: hidden` on `html` and `body`.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #11641

- Adds overflow hidden for html and body

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story.js [hunk 1]
```diff
@@ -295,6 +298,17 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /** @private */
+  lockBody_() {
+    const document = this.win.document;
+    setImportantStyles(document.documentElement, {
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.js]: Any reason we can't just add this to `amp-story.css` instead of adding it in the body?

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: actually i had a question regarding about this specific part, is there any plan at all where we allow `amp-story` without `standalone`? 

If no, css is much better.

If yes/maybe, a conditional check of `standalone` attribute might and js based solution might be better

**Comment by prateekbh** [comment]
will update with code for `standalone` checks

**Comment by prateekbh** [comment]
need someone to merge this

**Pull request merged**